tempo.yaml

```diff
@@ -0,0 +1,56 @@
+server:
+  http_listen_port: 3200
+
+distributor:
+  receivers:
+    otlp:
+      protocols:
+        grpc:
+        http:
+
+ingester:
+  trace_idle_period: 10s
+  max_block_duration: 5m
+
+compactor:
+  compaction:
+    block_retention: 1h
+
+metrics_generator:
+  processor:
+    local_blocks:
+      filter_server_spans: false
+      flush_to_storage: true
+    span_metrics:
+      dimensions:
+        - client
+        - result
+        - non_custody_indices
+        - imported_blocks
+        - missing_column_indexes
```

Comment:
These are the fields that gets added as labels in the generated metrics for query purpose.